backend/benefit/applications/api/v1/serializers.py

```diff
@@ -98,6 +99,13 @@ class Meta:
         ]
         read_only_fields = ["created_at"]
 
+    def validate(self, data):
+        if data["attachment_file"].size > MAX_UPLOAD_SIZE:
+            raise serializers.ValidationError(
+                f"Upload file size cannot be greater than {MAX_UPLOAD_SIZE} bytes"
```

Comment:
Should error messages like these be localized? I see that I've myself been inconsistent with this

Reply:
A separate jira issue was created for this yesterday